docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc

```diff
@@ -69,7 +69,7 @@ yq -i eval ".parameters.openshift4_terraform.terraform_variables.cluster_name =
 .For cloudscale.ch
 [source,bash]
 ----
-yq -i eval ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"plus-32-4\"" \
+yq -i eval ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"plus-24\"" \
```

Comment:
This seems unrelated, and IMO `plus-32-4` is the correct infra node flavor.

```suggestion
```

Reply:
Yes it is 🙈 Seemed to happen in thee rebase. Weird 😅